### PR TITLE
Add monitoring, observing, and tuning sections

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
### Description

Adds TOC changes for monitoring, tuning, and observability sections

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by ariamarble — APPROVED** [review]
LGTM, should be exciting :)

**Review by hdhalter — COMMENTED** [review]
In our TOC proposal, we have Dashboards, then Security, then Search, then Tuning your cluster. Do you plan to move Security and Search up above this?

File: _tuning-your-cluster/cluster.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ---
 layout: default
 title: Cluster formation
```
Comment (hdhalter) [inline _tuning-your-cluster/cluster.md:3]: Are we going with 'cluster formation' or should we rename to "Creating a cluster"?

File: _tuning-your-cluster/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+nav_order: 7
+has_children: false
+has_toc: true
+---
+
+# Tuning your cluster
+
+OpenSearch allows system administrations many options to optimize the performance of their OpenSearch cluster.
+
+- [Cluster formation]({{site.url}}{{site.baseurl}}/tuning-your-cluster/cluster/)
```
Comment (hdhalter) [inline _tuning-your-cluster/index.md:13]: This order doesn't match the TOC. Can we get an automated list here?

File: _observing-your-data/trace/get-started.md [hunk 3]
```diff
@@ -3,6 +3,8 @@ layout: default
 title: Get Started
```
Comment (hdhalter) [inline _observing-your-data/trace/get-started.md:3]: Getting started?

File: _tuning-your-cluster/replication-plugin/get-started.md [hunk 4]
```diff
@@ -1,7 +1,10 @@
...
 layout: default
 title: Get started
-nav_order: 10
+nav_order: 15
+parent: Cross-cluster replication
+redirect_from:
+  - /replication-plugin/get-started/
 ---
 
 # Get started with cross-cluster replication
```
Comment (hdhalter) [inline _tuning-your-cluster/replication-plugin/get-started.md:10]: Getting started...?

File: _tuning-your-cluster/replication-plugin/settings.md [hunk 5]
```diff
@@ -2,6 +2,9 @@
 layout: default
 title: Settings
```
Comment (hdhalter) [inline _tuning-your-cluster/replication-plugin/settings.md:3]: Replication settings

**Review by hdhalter — COMMENTED** [review]
A few comments. Other than that, looks great! Thanks, Nate.

**Pull request closed without merging**